loops/swift/code.swift

```diff
@@ -0,0 +1,12 @@
+let u = UInt(CommandLine.arguments[1])!  // Get input number from command line
+let r = UInt.random(in: 0..<10000)  // Get random number 0 <= r < 10k
+var a: [UInt] = Array(repeating: 0, count: 10000)  // Array of 10k elements initialized to 0
+
+for i in 0..<10000 {  // 10k outer loop iterations
+    for j in 0..<100000 {  // 100k inner loop iterations
+        a[i] += UInt(j) % u  // Simple sum
+    }
```

Comment:
Does type casting inside of the loop impact performance? Would it be better to change the for loop declaration to iterate over `UInt`s?

```suggestion
    for j in 0..<UInt(100000) {  // 100k inner loop iterations
        a[i] += j % u  // Simple sum
    }
```

Reply:
exactly my point here https://github.com/bddicken/languages/pull/18#issuecomment-2499392211

Reply:
Oh I must have started my review before you posted that comment. Sounds like we are in agreement then!